client/src/styles/global/_about.scss

```diff
@@ -0,0 +1,131 @@
+
+.about{
+  margin-top: 25px;
+
+}
+.title {
+  margin-bottom: 70px;
+}
+.developer {
+  display: flex;
+  background-color: rgba($color-text-inactive, 0.2);
+  padding: 0;
+  margin-bottom: 30px;
+  box-shadow: 7px 7px 36px -10px rgba(0, 0, 0, 0.25);
+  &__img {
+    height: 340px;
+    flex: 0 0 300px;
+  }
+  &__info {
+    padding: 20px 30px;
+    text-align: left;
+
+  }
+  &__name {
+    margin-bottom: 10px;
+    font-size: 28px;
+  }
+  &__role {
+    margin-bottom: 20px;
+  }
+  &__descr {
+    margin-top: 10px;
+    text-align: justify;
+    text-indent: 40px;
+    font-size: 16px;
+  }
+}
+#stas {
+  .developer__img{
+    background-image: url(../assets/kravchuk.jpg);
+    background-position: center 10%;
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+}
+
+#phil {
+  .developer__img{
+    background-image: url(../assets/khromov.jpg);
+    background-position: center center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    
+  }
+}
+
+#alina {
+  .developer__img{
+    background-image: url(../assets/khasanova.png);
+    background-position: center 30%;
+    background-size: 450px;
+    background-repeat: no-repeat;
+    
+  }
+}
+
+@media screen and (max-width: 980px) {
+  .title{
+    font-size: 28px;
+  }
+  .developer{
+    &__name{
+      font-size: 24px;
+    }
+    &__role{
+      font-size: 18px;
+    }
+    &__descr{
+      font-size: 14px;
+    }
+  }
+}
+
+@media screen and (max-width: 810px) {
+  .title{
+    font-size: 26px;
+  }
+  #stas{
+    .developer__img{
+      background-position: center center;
+    }
+  }
+  .developer{
+    &__img{
+      flex-basis: 250px;
+    }
+    &__name{
+      font-size: 20px;
+    }
+    &__role{
+      font-size: 16px;
+    }
+    &__descr{
+      font-size: 12px;
+      margin-top: 7px;
+    }
+  }
+}
+
+@media screen and (max-width: 660px) {
+  .title{
+    font-size: 26px;
+    margin-bottom: 40px;
+  }
+  .developer{
+    &__img{
+      display: none;
```

Comment:
I'm lacking photos in mobile version:(

Reply:
<img width="343" alt="image" src="https://user-images.githubusercontent.com/86129327/188501737-9f0ed0af-ea14-4acb-a629-a82d5e48eb63.png">

I think this is better

Reply:
👍🏻